data-reconciliation-app/Makefile

```diff
@@ -48,7 +48,7 @@ build-enclave: build ## 📦 Build Enclave container image from Dockerfile
 	@../scripts/build_image.sh enclave
 
 e2e-test: ## 📦 Run end to end tests
-	@echo -e "\e[34m Not Implemented \e[0m" || true
+	@cd ./test/e2e-test/ && npm install --force && npm run start
```

Comment:
To make sure the e2e testing app can be run independently, I have added `npm install --force`.

Reply:
There is a step that is run when the devcontainer is created. Maybe you could save some time here `/workspaces/ccf-app-samples/scripts/initialise_devContainer.sh`

Reply:
Updated.